src/NzbDrone.Core/Download/Clients/DownloadStation/DiskStationApiInfoCache.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Download.Clients.DownloadStation
+{
+    public class DiskStationApiInfoCache
```

Comment:
I'm wondering if we should just add Name, Type, and NeedsAuth directly to DiskStationApiInfo. That way it's entirely descriptive of the api.